.github/BOTMETA.yml

```diff
@@ -297,35 +297,37 @@ files:
   $modules/cloud/smartos/vmadm.py: jasperla
   $modules/cloud/softlayer/sl_vm.py: mcltn
   $modules/cloud/univention/: 2-B
-  $modules/cloud/vmware/: $team_vmware
+  $modules/cloud/vmware/:
+    maintainers: $team_vmware
+    ignored: jcpowermac mtnbikenc
   $modules/cloud/vmware/vca_fw.py: privateip
   $modules/cloud/vmware/vca_nat.py: privateip
   $modules/cloud/vmware/vca_vapp.py: privateip
   $modules/cloud/vmware/vcenter_license.py: dagwieers
-  $modules/cloud/vmware/vmware_cluster.py: $team_ansible jcpowermac
-  $modules/cloud/vmware/vmware_datacenter.py: $team_ansible jcpowermac kamsz
-  $modules/cloud/vmware/vmware_dns_config.py: $team_ansible jcpowermac
-  $modules/cloud/vmware/vmware_dvs_host.py: $team_ansible jcpowermac
-  $modules/cloud/vmware/vmware_dvs_portgroup.py: $team_ansible jcpowermac
-  $modules/cloud/vmware/vmware_dvswitch.py: $team_ansible jcpowermac
+  $modules/cloud/vmware/vmware_cluster.py: $team_ansible
+  $modules/cloud/vmware/vmware_datacenter.py: $team_ansible kamsz
+  $modules/cloud/vmware/vmware_dns_config.py: $team_ansible
+  $modules/cloud/vmware/vmware_dvs_host.py: $team_ansible
+  $modules/cloud/vmware/vmware_dvs_portgroup.py: $team_ansible
+  $modules/cloud/vmware/vmware_dvswitch.py: $team_ansible
   $modules/cloud/vmware/vmware_guest.py: dagwieers
   $modules/cloud/vmware/vmware_guest_facts.py: nerzhul
-  $modules/cloud/vmware/vmware_guest_find.py: $team_vmware
-  $modules/cloud/vmware/vmware_host.py: $team_ansible jcpowermac dagwieers
+  $modules/cloud/vmware/vmware_guest_find.py:
```

Comment:
I believe `$team_vmware` should still be responsible for this module.

Reply:
`$team_vmware` already applied to the entire `vmware/` directory and has been moved to `maintainers:` on line 301.

Reply:
Thanks for the clarification.